pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -69,6 +69,17 @@ type JobSpec struct {
 
 	// URL to the originator (eg, PR URL)
 	URL string `json:"url,omitempty"`
+
+	// Relevant options when the job is "export"
+	Export *ExportOptions `json:"export,omitempty"`
+}
+
+type ExportOptions struct {
```

Comment:
(likely not in this PR). What about given the option to pick the branch so that users can open a PR for this change? 

The system will create the branch out of the default branch, commit the necessary changes and push it to the remote. 
Once the job is completed, the user is present with an option to open a pull request.